--- ./debian/rules	2025-06-16 16:01:27.478961797 +0700
+++ ./debian/rules	2025-06-16 16:10:43.396229737 +0700
@@ -35,9 +35,9 @@
 DEB_CFLAGS_MAINT_APPEND += -DFIRMWARE_DIR=/lib/firmware
 export DEB_CFLAGS_MAINT_APPEND
 
-ifeq (,$(filter $(DEB_HOST_ARCH),i386 alpha m68k ppc64 sh4 x32))
-  CONFIGURE_FLAGS += --with-phonebook=ebook
-endif
+# ifeq (,$(filter $(DEB_HOST_ARCH),i386 alpha m68k ppc64 sh4 x32))
+#   CONFIGURE_FLAGS += --with-phonebook=ebook
+# endif
 
 ifeq ($(shell dpkg-vendor --is Ubuntu && echo yes) $(DEB_HOST_ARCH), yes i386)
    skip_packages = -Nbluez-cups

--- ./debian/changelog	2025-06-16 18:46:48.544507852 +0700
+++ ./debian/changelog	2025-06-16 18:46:35.320217008 +0700
@@ -1,3 +1,10 @@
+bluez (5.82-1.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix evolution dependency
+
+ -- Nikolay Bely <belyynikolay@gmail.org>  Mon, 16 Jun 2025 18:48:11 +0700
+
 bluez (5.82-1.1) unstable; urgency=medium
 
   * Non-maintainer upload.
