--- ./debian/rules	2025-06-16 16:01:27.478961797 +0700
+++ ./debian/rules-new	2025-06-16 16:10:43.396229737 +0700
@@ -35,9 +35,9 @@
 DEB_CFLAGS_MAINT_APPEND += -DFIRMWARE_DIR=/lib/firmware
 export DEB_CFLAGS_MAINT_APPEND
 
-ifeq (,$(filter $(DEB_HOST_ARCH),i386 alpha m68k ppc64 sh4 x32))
-  CONFIGURE_FLAGS += --with-phonebook=ebook
-endif
+# ifeq (,$(filter $(DEB_HOST_ARCH),i386 alpha m68k ppc64 sh4 x32))
+#   CONFIGURE_FLAGS += --with-phonebook=ebook
+# endif
 
 ifeq ($(shell dpkg-vendor --is Ubuntu && echo yes) $(DEB_HOST_ARCH), yes i386)
    skip_packages = -Nbluez-cups
